opentasks/src/main/java/org/dmfs/tasks/utils/SearchChildDescriptor.java

```diff
@@ -58,7 +58,7 @@ public SearchChildDescriptor(String authority, String queryColumn, String[] proj
 
     /**
      * Get a new {@link CursorLoader} and update it's selection arguments with the values in {@code cursor} as defined by {@code selectionColumns} in
-     * {@link #ExpandableChildDescriptor(Uri, String[], String, String, int...)}. Also applies any selection defined by <code>filter</code>.
+     * . Also applies any selection defined by <code>filter</code>.
```

Comment:
The javadoc is broken here. Not sure what this was supposed to mean. Maybe just remove the entire thing. Better have no documentation than an incorrect one.